beacon-chain/sync/initial-sync/round_robin.go

```diff
@@ -44,15 +45,20 @@ func (s *Service) roundRobinSync(genesis time.Time) error {
 	counter := ratecounter.NewRateCounter(counterSeconds * time.Second)
 	randGenerator := rand.New(rand.NewSource(time.Now().Unix()))
 	var lastEmptyRequests int
+	highestFinalizedSlot := helpers.StartSlot(s.highestFinalizedEpoch() + 1)
 	// Step 1 - Sync to end of finalized epoch.
-	for s.chain.HeadSlot() < helpers.StartSlot(s.highestFinalizedEpoch()+1) {
+	for s.chain.HeadSlot() < highestFinalizedSlot {
```

Comment:
the result of this change can be great . as it stays the same for all the hours of initial sync. 
the result as i see it will be that the best peer will be requested for all epochs from this initial value that was set at the beginning of sync to the new finalised epoch that is the current one  (maybe a day later). this can put a lot of stress on one peer. 
maybe this can be done to resolve this issue?
```suggestion
	for s.chain.HeadSlot() < max(highestFinalizedSlot,helpers.StartSlot(s.highestFinalizedEpoch() + 1)) {
```

Reply:
it doesn't stay the same, we just ensure there are a safe amount of peers before updating it . Below there is this piece of code: 

```go
	if len(peers) >= flags.Get().MinimumSyncPeers {
			highestFinalizedSlot = helpers.StartSlot(finalizedEpoch + 1)
		}
```

Reply:
missed it. thanks